bokeh/server/views/auth_request_handler.py

```diff
@@ -18,12 +18,16 @@
 # Imports
 #-----------------------------------------------------------------------------
 
+# Bokeh imports
+#External imports
+from tornado.web import RequestHandler
```

Comment:
You need to run trivial tests like codebase tests locally, so that you don't eat up our testing bandwidth, especially if there will be multiple people pushing stuff simultaneously to github:
```
pytest tests/codebase
```

Reply:
Thanks, I will do the same from now on. There was a minor isort issue which I could not catch locally that's why the pr failed.

Reply:
isort is pretty fragile when it comes to external dependencies, which if not installed, it eagerly considers as internal modules. A temporary workaround is to work in a conda environment with all dependencies installed, e.g.
```
conda create -n my_env python=3.8 $(python scripts/deps.py)`
conda activate my_env
```